### ISSUE TITLE
Remove trailing and leading whitespace in A11y strings

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#137]
Per https://github.com/phetsims/rosetta/issues/193#issuecomment-518865992, strings with trailing or leading whitespace should be trimmed and the whitespace should be placed into larger patterns.
